helm-chart-sources/pulsar/values.yaml

```diff
@@ -1070,6 +1076,11 @@ brokerSts:
     usePolicy: true
     maxUnavailable: 1
 
+## Pulsar Transactions metadata. The metadata will be deployed as
```

Comment:
we can clean up our wording here, you don't "deploy metadata".

what about:

"Pulsar Transactions metadata. Metadata will be initialised as soon as one broker pod is available"